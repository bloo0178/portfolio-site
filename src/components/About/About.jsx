import React from "react";
import styles from "./About.module.scss";

class About extends React.Component {
  render() {
    return (
      <div className={styles.container}>
        <h2>About</h2>
        <p>
          Lorem ipsum dolor sit amet, congue tincidunt cu nec, diam exerci id
          sed. Ne pri errem alterum, ne quis quaerendum est, dico inimicus his
          te. Fierent urbanitas vel te, in soleat commune tacimates est. Solum
          populo dissentiet sed et, eu fugit oblique eam. Nullam quodsi
          appellantur ad usu.
        </p>
        <p>
          {" "}
          Id sea libris virtute, case rationibus elaboraret ut eum. Eum harum
          salutandi no. Eu his nulla omnes, has quidam facete eu. Ne mea odio
          minim, nobis corpora ei mea. Qui lorem officiis ea. Novum homero
          appareat ea eam, per porro altera volutpat eu.
        </p>
        <p>
          {" "}
          Quo alii voluptua apeirian in. Ex modus noster nostrum sed. Qui no
          paulo platonem, has te ridens praesent. Eam atomorum mandamus
          expetenda cu.
        </p>
      </div>
    );
  }
}

export default About;
