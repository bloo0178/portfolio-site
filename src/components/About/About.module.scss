@import "../../common/_variables";

.container {
  background-color: transparent;
  display: flex;
  flex-direction: column;
  align-items: center;
  padding-top: 10%;
  padding-bottom: 10%;
  padding-left: 10%;
  padding-right: 10%;
  //text-align: center;
}

.content {
    align-self: left;
}