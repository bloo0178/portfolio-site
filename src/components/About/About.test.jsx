import React from "react";
import { create } from "react-test-renderer";
import { About } from "./About";

describe("Card component", () => {
	test("it matches the snapshot", () => {
		const component = create(<About />);
		expect(component.toJSON()).toMatchSnapshot();
	});
});
