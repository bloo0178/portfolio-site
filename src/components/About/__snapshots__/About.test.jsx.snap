// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component it matches the snapshot 1`] = `
<div
  className="container"
>
  <div
    className="content"
  >
    <h1>
      About
    </h1>
    <p>
      I am fascinated with the infinite challenge of learning and applying technology to evolve the things we do. My passion is software development, grounded in over a decade of professional experience applying systems and design thinking.
    </p>
    <p>
      I have worked between businesses and users throughout my career - defining how to use technology to support those relationships. Most recently as a Solutions Architect at an award-winning digital business consultancy, I collaborated with UX designers and developers to create digital strategies paired with custom software solutions.
    </p>
    <p>
      I currently work heavily with React for interface development and have been using serverless approaches for backend needs. I am not rigid in my technology choices and refresh my knowledge regularly.
    </p>
    <p>
      I am interested in opportunities that involve solving practical business or systemic problems through focused analysis, system design, and ultimately directed software development. If my background interests you, let's connect and see how we can work together.
    </p>
  </div>
</div>
`;
